tests/lib/rules/define-props-declaration.js

```diff
@@ -117,13 +121,42 @@ tester.run('define-props-declaration', rule, {
       })
       </script>
       `,
+      output: `
+      <script setup lang="ts">
+      const props = defineProps<{ kind: string }>()
```

Comment:
If `required` is not set (or for shorthand prop types), shouldn't this result in optional props (i.e. `kind?: string`)?

Reply:
You're right. Fixed.